### PR TITLE
Add International Image Interoperability Framework

### PR DESCRIPTION
**Pull request by jronallo**
**https://github.com/IIIF/awesome-iiif**

The International Image Interoperability Framework is a group of standard APIs around sharing and reuse of media. It is also a growing community of galleries, libraries, archives, museums, companies, and others who develop the standards and interoperable implementations. Content includes helpful links around each of the standards, demonstrations of their use, and tutorials and presentations. The list is especially helpful for orienting new community members and developers.
# ALL THE BELOW CHECKBOXES SHOULD BE DONE AND CHECKED AND THE ABOVE PLACEHOLDERS SHOULD BE FILLED IN BEFORE SUBMITTING A PULL REQUEST!
- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title. _(For example: `Add Node.js`)_
- The list I added
  - [x] **has been around for at least 20 days,**
  - [x] is a non-generated Markdown file in a GitHub repo,
  - [x] is not a duplicate,
  - [x] only has awesome items with descriptions _(Awesome lists are curations of the best, not of everything)_,
  - [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
  - [x] has a Table of Contents section named `Contents` as the first section,
  - [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) _(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)_,
  - [x] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
  - [ ] has been checked for inconsistent formatting, spelling, and grammar _(This includes starting each link description with an uppercase character and ending it with a period `.`, and consistent naming)_,
  - [x] has been added at the bottom of the appropriate category.

…/IIIF/awesome-iiif

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -337,6 +337,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Open Source Documents](https://github.com/hubtee/awesome-opensource-documents)
 - [Audio Visualization](https://github.com/willianjusten/awesome-audio-visualization)
 - [Broadcasting](https://github.com/ebu/awesome-broadcasting)
+- [International Image Interoperability Framework (IIIF)](https://github.com/IIIF/awesome-iiif) - standard APIs for sharing and presenting media
```
Comment (sindresorhus) [inline readme.md]: Drop the `(IIF)` and the description needs to follow the format like the others.

**Comment by sindresorhus** [comment]
Feedback:

- I've unchecked one checkbox that is clearly not done.
- Heading: `Awesome IIIF` => `Awesome International Image Interoperability Framework`
- Include project logo in the top right, like e.g. https://github.com/sindresorhus/awesome-electron (The logo should link to the project website) (The logo should be high-DPI, set it to minimum half the width of the original image)
- There should be a succinct summary at the top about the project. (You could use the one in this PR)
- I would like to see some links to community resources (Example: https://github.com/sindresorhus/awesome-electron#community).
- Maybe include links to videos of talks and other things related.

**Comment by sindresorhus** [comment]
ping

**Comment by jronallo** [comment]
Just got back from a IIIF working group meeting. Part of the discussion was about documentation and our awesome list. We have a group forming to maintain the list, so I expect we'll get back to resolving the issues you pointed out soon.

**Comment by davisonio** [comment]
hey @jronallo any update on the feedback mentioned above?

**Comment by sindresorhus** [comment]
ping

**Comment by jronallo** [comment]
We've just now made several changes based on your feedback above: https://github.com/sindresorhus/awesome/pull/780#issuecomment-251878068

I still need to review the list for any consistency and update the pull request. In other words still working on this.

**Comment by davisonio** [comment]
Any more news on this? The PR needs updating with a removal of (IIIF) and description formatting. Else the recent changes in the repo look good 👍

**Comment by jronallo** [comment]
Now updated for requested changes to the link text and description formatting.

**Comment by sindresorhus** [comment]
> This includes starting each link description with an uppercase character and ending it with a period .

Not followed. Entries should have the format: `[Title](URL) - Description.`

**Comment by sindresorhus** [comment]
`#curators_of_awesome channel in Slack` in https://github.com/IIIF/awesome-iiif/blob/master/curation.md should be linkified to the Slack channel.

Otherwise looks pretty good :)

**Comment by sindresorhus** [comment]
Ping

**Comment by sindresorhus** [comment]
Closing for lack of response. There's no point in resubmitting at a later point as you've lost the chance to be included in the Awesome list.

**Pull request closed without merging**